### PR TITLE
Extend decoded values + HCA

### PR DESCRIPTION
**Pull request by jedi7**
* add support for BCD value decoding and 64bit values
* add support for H.C.A
* add support for OnTime and Operation Time

### CODE REVIEW

File: src/devices/m_bus.c [hunk 1]
```diff
@@ -310,7 +330,7 @@ static data_t *append_val(data_t *data, enum UnitType unit_type, uint8_t value_t
         fprintf(stderr, "M-Bus: Program error, exp (%d) is out of bounds", exp);
         return data;
     }
-    float fvalue = val * pow10_table[exp];
+    double fvalue = val * pow10_table[exp];
```
Comment (merbanan) [inline src/devices/m_bus.c:333]: Is double really needed?

Comment (jedi7) [reply src/devices/m_bus.c:333]: Not fully sure. But I increased it because we have here 64bit values. So to keep precision.

Comment (zuckschwerdt) [reply src/devices/m_bus.c:333]: This does not do what one might think. The calculation ist float, the float is then stored as double.

Comment (jedi7) [reply src/devices/m_bus.c:333]: you are right, I forgot to change also pow10_table

**Pull request merged**